modules/ROOT/pages/metrics/coding.adoc

```diff
@@ -0,0 +1,16 @@
+include::{partialsdir}/attributes.adoc[]
+
+include::configuring-dev-env-metrics.adoc[]
+
+== Setting up {metrics-service} service SDK
+
+The {metrics-service} service SDK is included by default when installing any of the components from {SDK}, so Metrics are sent automatically from the mobile app.
```

Comment:
I think this needs to be more explicit.

I like the 'Configuring your Development Environment for the Push Notifications Service' heading followed by:
* download json
* add library from AeroGear SDK
as  a flow

The alternative approach is in keycloak
* download json
* Add 'auth' to your app

whichever path you go, the instructions should be clear, eg:

. If you already use a component from the {SDK} in your app, you can skip this procedure.
. Install the core component
. If you are developing a Cordova app ....